Makefile

```
@@ -26,3 +26,14 @@ run: build
 		-m 1G \
 		-kernel kernel-$(ARCH).bin
 
+build-test:
+	cd test-runner && cargo build --target $(TARGET)
+
+test: build-test
```

checkmake (0.2.2)

[warning] Target body for "test" exceeds allowed length of 5 (7).

(maxbodylength)